gcp/table_gcp_storage_bucket_object.go

```diff
@@ -0,0 +1,349 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/storage/v1"
+)
+
+func tableGcpBucketStorageObject(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_storage_bucket_object",
+		Description: "GCP Storage Object",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"bucket", "name"}),
+			Hydrate:    getBucketStorageObject,
+		},
+		List: &plugin.ListConfig{
+			KeyColumns:    plugin.OptionalColumns([]string{"bucket"}),
+			ParentHydrate: listGcpStorageBuckets,
```

Comment:
I wouldn't use a parent hydrate for this table, as there could be a lot of buckets with a lot of objects, so if someone runs `select * from gcp_storage_object`, they could be waiting a long time (we made the same decision with `aws_s3_object`.